### PR TITLE
[File Explorer Source Control Integration] Accessibility bug fixes

### PR DESCRIPTION
**Pull request by ssparach**
## Summary of the pull request
This PR fixes two accessibility bugs for the File Explorer Page under Windows Customization. 

## Detailed description of the pull request / Additional comments
The accessibility bugs that this PR resolves are: 
- On navigating to 'Unassigned' button (dropdown) or 'More options' button, narrator is not announcing position count for the list items. Arrow navigation is defined for 'Folder paths' so position count should be defined.
- On navigating to 'Unassigned' button (dropdown), narrator is announcing as 'Unassigned button collapsed' but it is not announcing its respective 'Folder path'.

## Validation steps performed
Used narrator to confirm that the following expected results were achieved: 
- Position count defined on navigating to the dropdown button or 'More options' button i.e. narrator announces if an item is one of two, two of four etc. 
- Narrator announces the associated folder path info on navigating to dropdown button 

## PR checklist
- [ ] Closes #xxx
- [ ] Tests added/passed
- [ ] Documentation updated

### CODE REVIEW

File: tools/Customization/DevHome.Customization/Views/AddRepositoriesView.xaml [hunk 1]
```diff
@@ -40,15 +40,20 @@
                             Orientation="Horizontal">
                             <DropDownButton 
                                 Content="{Binding SourceControlProviderDisplayName}"
-                                ToolTipService.ToolTip="{Binding SourceControlProviderPackageDisplayName}">
+                                AutomationProperties.Name="{Binding RepoPathMapping}"
```
Comment (krschau) [inline tools/Customization/DevHome.Customization/Views/AddRepositoriesView.xaml:43]: Does this set the name correctly (does it sound right in Narrator / look right in acc insights/inspect) or don't you need to do the 
`<data name="ButtonName.[using:Microsoft.UI.Xaml.Automation]AutomationProperties.Name" xml:space="preserve">` thing in the resw file?

Comment (ssparach) [reply tools/Customization/DevHome.Customization/Views/AddRepositoriesView.xaml:43]: Yes, it does seem to set the name correctly. In narrator, the path is correctly announced with backslashes, followed by the status of the drop down button. I also checked with Accessibility Insights and it did not flag any issues.  Thanks!

File: tools/Customization/DevHome.Customization/Models/RepositoryInformation.cs [hunk 2]
```diff
@@ -21,13 +20,22 @@ public class RepositoryInformation
 
     public string SourceControlProviderPackageDisplayName { get; }
 
-    public RepositoryInformation(string rootpath, string classId)
+    public string RepoPathMapping { get; }
```
Comment (dhoehna) [inline tools/Customization/DevHome.Customization/Models/RepositoryInformation.cs:23]: Please use `Repository` and not `Repo`.  I want DevHome to stay consistent in regards to that word.

File: tools/Customization/DevHome.Customization/ViewModels/FileExplorerViewModel.cs [hunk 3]
```diff
@@ -70,9 +70,11 @@ public void RefreshTrackedRepositories()
         {
             TrackedRepositories.Clear();
             var repoCollection = RepoTracker.GetAllTrackedRepositories();
+            var position = 0;
             foreach (KeyValuePair<string, string> data in repoCollection)
             {
-                TrackedRepositories.Add(new RepositoryInformation(data.Key, data.Value));
+                position++;
```
Comment (dhoehna) [inline tools/Customization/DevHome.Customization/ViewModels/FileExplorerViewModel.cs:76]: NIT: Change to a `for` loop that keeps track of the position.

File: tools/Customization/DevHome.Customization/Views/AddRepositoriesView.xaml (hunk 1 above)
Comment (dhoehna) [reply tools/Customization/DevHome.Customization/Views/AddRepositoriesView.xaml:43]: Setting a binding for the automation name is needed when something has to change between all the items.  For example, in RepoConfig.  The narrator says for the edit button "Edit repository [path]" where path is different for each item.

**Review by dhoehna — APPROVED** [review]

File: tools/Customization/DevHome.Customization/Models/RepositoryInformation.cs (hunk 2 above)
Comment (ssparach) [reply tools/Customization/DevHome.Customization/Models/RepositoryInformation.cs]: Modified in latest. Thanks!

**Review by krschau — APPROVED** [review]

**Pull request merged**